library/Rules/LanguageCode.php

```diff
@@ -0,0 +1,564 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+use Respect\Validation\Exceptions\ComponentException;
+
+/**
+ * Validates countries in ISO 639.
```

Comment:
Validates _languages_ in ISO 639.

PS.: That's one of the reasons why I hate documentation on code... The code must explain itself, but since we're an open source library, that kinda have to do this... Or not?